index.html

```diff
@@ -39,8 +39,161 @@ <h1>I’m Abhishek <br>
+                        <li>Backend Dev</li>
+                        <li><img src="assets/images/Counter.png" alt=""></li>
+                        <li>2015</li>
+                    </ul>
+                    </div>
+                <p class="para">A daily selection of privately<br> personalized
+                    reads; no accounts or<br> sign-ups required.</p>
+                    <div class="button-list">
+                        <ul>
+                          <li><a href="">html</a></li>
+                          <li><a href="">css</a></li>
+                          <li><a href="">javaScript</a></li>
+                        </ul>
+                      </div>
+                  <div> 
+                <button class="btn" type="button">See Project</button>
+                </div>
+            </div>
+        </div>
+    <div class="describe">
+        <h3>About <br> Myself</h3>
+        <p class="description">Hello I’m a software
+            developer! I can help<br> you build a product ,
+            feature or website<br> Look through some of my work
+            and<br> experience! If you like what you see and<br>
+            have a project you need coded, don’t<br> hestiate to
+            contact me.</p>
+        <div class="connect"><a href="#" class="connect">LET'S CONNECT</a></div>
+        <div class="contacts">
+            <ul class="social-icon">
+                <li>
+                    <a href="#" class="link"><img src="assets/images/Vector (2).svg" alt="">
+                            </a>
+                </li>
+                <li><a href="#" class="link"><img src="assets/images/Linkedin.svg" alt=""></a></li>
+                <li><a href="#" class="link"><img src="assets/images/Vector (3).svg" alt=""></a></li>
+                <li><a href="#" class="link"><img src="assets/images/Vector.svg" alt=""></a>
+                    <li><a href="#" class="link">
+                        <img src="assets/images/Vector (1).svg" alt="">
+                        <!-- <i class="fab fa-medium-m"></i> -->
+                        </a>
+                    </li>
+                </ul>
+            </div>
+
+            <button class="press" type="button">Get Resume</button>
+
+            <div class="skills">
+                <p class="skill"> Languages</p>
+                <ul class="languages">
+                    <li><img src="assets/images/Ellipse1.png" alt=""><a
+                            href="#" class="tech">JavaScript</a></li>
+                    <li> <img src="assets/images/Ellipse1-1.png" alt=""><a
+                            href="#" class="tech">HTML5</a></li>
+                    <li><img src="assets/images/Ellipse1-2.png" alt=""><a
+                            href="#" class="tech">CSS</a></li>
+                </ul>
+
+                <p class="skill">Frameworks</p>
+                <hr>
+                <p class="skill">Skills</p>
+               <hr>
+
+            </div>
```

Comment:
The languages, frameworks and skills should have the arrows as shown in the figma designs below. Kindly check and implement that. 

![Screenshot from 2022-09-23 19-16-12](https://user-images.githubusercontent.com/65233860/192041194-5e35ba7b-2910-41be-91af-e5d80a2bf614.png)